<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ECGSettingsWidget</class>
 <widget class="QWidget" name="ECGSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="FilePickerWidget" name="cardioPath" native="true"/>
   </item>
   <item>
    <widget class="FilePickerWidget" name="initPath" native="true"/>
   </item>
   <item>
    <widget class="FilePickerWidget" name="databasePath" native="true"/>
   </item>
   <item>
    <widget class="FilePickerWidget" name="exportPath" native="true"/>
   </item>
   <item>
    <widget class="FilePickerWidget" name="settingsFileName" native="true"/>
   </item>
   <item>
    <widget class="FilePickerWidget" name="winSettingsFileName" native="true"/>
   </item>
   <item>
    <widget class="FilePickerWidget" name="btrRecordFileName" native="true"/>
   </item>
   <item>
    <widget class="FilePickerWidget" name="btrDatabaseFileName" native="true"/>
   </item>
   <item>
    <widget class="FilePickerWidget" name="xmlFileName" native="true"/>
   </item>
   <item>
    <widget class="FilePickerWidget" name="pdfFileNameRestingEcg" native="true"/>
   </item>
   <item>
    <widget class="FilePickerWidget" name="pdfFileNameFullEcg" native="true"/>
   </item>
   <item>
    <widget class="FilePickerWidget" name="executableForParticipantInfo" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilePickerWidget</class>
   <extends>QWidget</extends>
   <header>widgets/file_picker_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
